docs/resources/User.md

```diff
@@ -28,7 +28,8 @@ There are other rules and restrictions not shared here for the sake of spam and
 | ------------- | --------- | ---------------------------------------------------------------------------------------------------- | --------------------- |
 | id            | snowflake | the user's id                                                                                        | identify              |
 | username      | string    | the user's username, not unique across the platform                                                  | identify              |
-| discriminator | string    | the user's 4-digit discord-tag                                                                       | identify              |
```

Comment:
You seem to have missed [this](https://discord.com/developers/docs/resources/user#usernames-and-nicknames) with the restrictions (or link) at the header `New Usernames` [here](https://support.discord.com/hc/en-us/articles/12620128861463).